src/index.js

```diff
@@ -474,7 +475,7 @@ function Carotte(config) {
         const exchangeName = getExchangeName(options);
         const queueName = getQueueName(options, config);
 
-        return Promise.race([
+        return Promise.race(promises.concat([
```

Comment:
Hum I think this is a breaking change. Previously the function was returning a `Promise<ActualQueue>` and is not returning a `Promise<ActualQueue | DescribeQueue>`.

Reply:
You're right, will be better to just wait for it but still return the actual queue

Reply:
Agreed, I was confused as of why we raced this promise with the other ones (timeout and "subscribeToActualQueue"). Just awaiting it should work just as well, right?